confd/production.tmpl

```diff
@@ -26,6 +26,13 @@
         "multithread": {{ getv "/configuration/exporter/worker/gdal/multithread" "true" }},
         "thread_count": {{ getv "/configuration/exporter/worker/gdal/thread/count" "-1" }}
     },
+    "s3": {
+        "endpoint_url": "{{ getv "/configuration/exporter/worker/s3/endpoint/url" "http://localhost:9000" }}",
+        "access_key_id": "{{ getv "/configuration/exporter/worker/s3/access/key/id" "minioadmin" }}",
+        "secret_access_key": "{{ getv "/configuration/exporter/worker/s3/secret/access/key" "minioadmin" }}",
+        "bucket": "{{ getv "/configuration/exporter/worker/s3/bucket" "mapping-images" }}"
+    },
+    "storage_provider": "{{ getv "/configuration/exporter/worker/storage/provider" "fs" }}",
```

Comment:
insert "external_physical_path" and "internal_outputs_path"  under fs {} config object

Reply:
rename "input_output" to something meaningful